morebits.js

```diff
@@ -3298,6 +3267,63 @@ Morebits.wiki.page = function(pageName, currentAction) {
 
 	};
 
+	// Common checks for action methods
+	// Used for move, undelete, delete, protect, stabilize
+	var fnPreflightChecks = function(action, onFailure) {
+		// if a non-admin tries to do this, don't bother
+		if (!Morebits.userIsSysop && action !== 'move') {
+			ctx.statusElement.error('Cannot ' + action + 'page : only admins can do that');
+			onFailure(this);
```

Comment:
pretty sure `this` here would be `window`. Can be fixed by changing the caller per prev cmt.